filer/private/sass/components/_filter-files.scss

```diff
@@ -4,22 +4,27 @@
 .filter-files-container {
     @include clearfix;
     position: relative;
-    float: left;
+    float: right;
+    width: calc(60% - 420px);
```

Comment:
its the size of action links and upload btns together. To use full other space for search